tests/table-operations.spec.ts

```diff
@@ -0,0 +1,495 @@
+  });
+
+  await test.step('Move Selection to Top', async () => {
+    await page.getByRole('button', { name: 'Move selection to top' }).click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Sort Descending', async () => {
+    await page.getByRole('button', { name: 'Sort descending' }).click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Sort Ascending', async () => {
+    await page.getByRole('button', { name: 'Sort ascending' }).click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Hide Selected', async () => {
+    await page.getByRole('button', { name: 'Hide Selected' }).click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Reset', async () => {
+    await page.getByRole('button', { name: 'Reset' }).click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  const dropIndicator = page
+    .locator('.visibility-ordering-list')
+    .locator('.marching-ants');
+  await test.step('Drag', async () => {
+    const floatOption = page.getByRole('button', {
+      name: 'Toggle visibility Float',
+    });
+    const stringOption = page.getByRole('button', {
+      name: 'Toggle visibility String',
+    });
+    await dragComponent(floatOption, stringOption, dropIndicator, 20);
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Create Group', async () => {
+    const stringOption = page.getByRole('button', {
+      name: 'Toggle visibility String',
+    });
+    await stringOption.click();
+    await page.getByRole('button', { name: 'Group' }).click();
+    await page.getByPlaceholder('Group Name').click();
+    await page.keyboard.type('test');
+    await page.keyboard.press('Enter');
+
+    const longOption = page.getByRole('button', {
+      name: 'Toggle visibility Long',
+    });
+    await dragComponent(longOption, stringOption, dropIndicator);
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+
+    // Edit Group Name
+    await page.getByRole('button', { name: 'Edit', exact: true }).click();
+    await page.keyboard.type('new_test');
+    await page.keyboard.press('Enter');
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+
+    // Delete Group
+    await page
+      .getByRole('button', { name: 'Delete group', exact: true })
+      .click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+
+  await test.step('Toggle Visibility', async () => {
+    await page
+      .getByRole('button', { name: 'Toggle visibility Double' })
+      .getByRole('button', { name: 'Toggle visibility' })
+      .click();
+
+    await expect(page.locator('.iris-grid-column')).toHaveScreenshot();
+  });
+});
+
+test('can custom column', async ({ page }) => {
```

Comment:
I'd just drop the `can` part of these tests - saying "can custom column" doesn't really make sense, it'd make more sense saying `test('custom columns'` or something like that.